### PR TITLE
Modifies the vTec cyborg upgrade to be toggleable between off/half/full

### PR DESCRIPTION
**Pull request by PolymorphBlue**
Adds a set of chevron icons to icons/mob/actions.dmi to show this - provided by Toriate

Makes this modification because a few different borgs stated that vTec could be hard to control, so makes it have a half-power feature.

### CODE REVIEW

**Review by deathride58 — CHANGES_REQUESTED** [review]
Needs to be modularized. The replacement of `R.speed = -2` with `R.AddAbility(...)` can stay in the file, but with a `//Citadel change - makes vtecs give an ability rather than reducing the borg's speed instantly` added onto it

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -79,7 +79,40 @@
 			to_chat(user, "<span class='notice'>There's no room for another VTEC unit!</span>")
 			return FALSE
 
-		R.speed = -2 // Gotta go fast.
+        //Citadel change - makes vtecs give an ability rather than reducing the borg's speed instantly
+		R.AddAbility(new/obj/effect/proc_holder/silicon/cyborg/vtecControl)
+
+
+/obj/effect/proc_holder/silicon/cyborg/vtecControl
```
Comment (jakeramsay007) [inline code/game/objects/items/robot/robot_upgrades.dm:86]: This proc can be modularised into modular_citadel

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 2]
```diff
@@ -79,7 +79,40 @@
...
+	name = "vTec Control"
+	desc = "Allows finer-grained control of the vTec speed boost."
+	action_icon = 'icons/mob/actions.dmi'
+	action_icon_state = "Chevron_State_0"
+
+	var/currentState = 0
+	var/maxReduction = 2
+
+
+/obj/effect/proc_holder/silicon/cyborg/vtecControl/Click(mob/living/silicon/robot/user)
```
Comment (jakeramsay007) [inline code/game/objects/items/robot/robot_upgrades.dm:96]: As can this one

**Review by jakeramsay007 — CHANGES_REQUESTED** [review]

**Review by deathride58 — APPROVED** [review]
We don't really have to worry about modularization anymore, since the mirror bot's turned off. So this can be merged

**Pull request merged**